README.md

```diff
@@ -430,17 +428,9 @@ await octokit.request("POST /repos/{owner}/{repo}/issues", {
 });
 ```
 
-The request is authenticated, retried, and throttled transparently by the `octokit` instance. The `accept` and `user-agent` headers are set as well.
-
 The 1st argument is the REST API route as listed in GitHub's API documentation. The 2nd argument is an object with all parameters, independent of whether they are used in the path, query, or body.
 
-See the standalone [`@octokit/request`](https://github.com/octokit/request.js#readme) module for more details on the `.request()` APIs.
-
-If you only need the request options, but want to send the actual request using your own method, use `octokit.request.endpoint(route, parameters)`. See the standalone [`@octokit/endpoint`](https://github.com/octokit/endpoint.js#readme) module for more details on the `.endpoint()` APIs.
```

Comment:
Didn't follow this but it seems "advanced" and fine for folks to discover reading the more detailed doc. If it is important to have here, then we likely need more supporting text to explain the scenario/use case.